pupil-spa/src/app/services/audit/audit.service.spec.ts

```diff
@@ -1,81 +1,40 @@
 import { TestBed } from '@angular/core/testing';
-import { StorageServiceMock } from '../storage/storage.service.mock';
 import { AuditService } from './audit.service';
 import { StorageService } from '../storage/storage.service';
-import { AuditEntry, QuestionRendered, CheckStarted, QuestionAnswered } from './auditEntry';
+import { QuestionRendered } from './auditEntry';
 
 let service: AuditService;
-let mockStorageService: StorageServiceMock;
+let storageService: StorageService;
 
 describe('AuditService', () => {
   beforeEach(() => {
-    mockStorageService = new StorageServiceMock();
+    storageService = new StorageService();
     const injector = TestBed.configureTestingModule({
       providers: [
         AuditService,
-        { provide: StorageService, useValue: mockStorageService }
+        StorageService
       ]
     });
     service = injector.get(AuditService);
+    storageService = injector.get(StorageService);
   });
 
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
 
   describe('addEntry', () => {
-
-    it('should add entry using audit key to storageService', () => {
-      spyOn(mockStorageService, 'setItem');
-
+    // it('should add entry to local storage with a unique key name', () => {
```

Comment:
Why is this commented out?

Reply:
ah thanks for the reminder. this test is redundant as the method no longer takes an argument.